src/libLookup/Lookup.cpp

```diff
@@ -64,6 +64,58 @@ Lookup::Lookup(Mediator& mediator) : m_mediator(mediator) {
 
 Lookup::~Lookup() {}
 
+void Lookup::Init() {
```

Comment:
Maybe change to a proper name rather than "init()", because this function is not supposed to be called in normal procedure.

Reply:
Fixed